task_11/src/img/icons/check-list.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
I suggest you to use [svgo](https://github.com/svg/svgo) to keep files more clean, add a little bit of performance. Just once with CLI when download something new: ```$ svgo -f . -o .```

Reply:
I didn't think about the optimization of svg before, thank for recommendation. Done.